test/e2e/operator_test.go

```diff
@@ -0,0 +1,61 @@
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	configv1 "github.com/openshift/api/config/v1"
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+
+	ntoclient "github.com/openshift/cluster-node-tuning-operator/pkg/client"
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestOperatorAvailable(t *testing.T) {
+	cl := ntoclient.GetClient()
```

Comment:
I like it better as you have it above `client := ntoclient.GetClient()`, no need to save a few characters.

Reply:
Changed as requested.